middleware/auth.js

```diff
@@ -0,0 +1,10 @@
+export default defineNuxtRouteMiddleware(async () => {
```

Comment:
Add the route members/[id] to the list that checks if the user is auth or not. 
For the profile page, if the user is not authenticated just show the regular page without the edit page options.